### PR TITLE
Enforce standard

### PR DESCRIPTION
**Pull request by Uzlopak**
This PR consists of three commits:

1. activate standard in project configuration
2. run the standard fixer
3. add button to Readme.md

#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: deps/dicer/lib/Dicer.js [hunk 1]
```diff
@@ -1,240 +1,206 @@
...
+  this._pause = false
+
+  this._hparser = new HeaderParser(cfg)
+  this._hparser.on('header', function (header) {
+    self._inHeader = false
+    self._part.emit('header', header)
+  })
 }
-inherits(Dicer, WritableStream);
+inherits(Dicer, WritableStream)
```
Comment (kibertoad) [inline deps/dicer/lib/Dicer.js:48]: we should try rewriting to classes and measure performance

File: deps/dicer/lib/HeaderParser.js [hunk 2]
```diff
@@ -1,108 +1,95 @@
...
-    len = lines.length;
-  let m, h;
+  const lines = this.buffer.split(RE_CRLF)
+  const len = lines.length
+  let m, h
 
-  for (var i = 0; i < len; ++i) {
-    if (lines[i].length === 0)
-      continue;
+  for (let i = 0; i < len; ++i) {
```
Comment (kibertoad) [inline deps/dicer/lib/HeaderParser.js:73]: the usual suspect :)

File: deps/streamsearch/sbmh.js [hunk 3]
```diff
@@ -200,33 +194,31 @@ SBMH.prototype._sbmh_feed = function (data) {
...
-    : data[pos];
-};
+    : data[pos]
+}
 
 SBMH.prototype._sbmh_memcmp = function (data, pos, len) {
-  for (var i = 0; i < len; ++i) {
-    if (this._sbmh_lookup_char(data, pos + i) !== this._needle[i])
-      return false;
+  for (let i = 0; i < len; ++i) {
```
Comment (kibertoad) [inline deps/streamsearch/sbmh.js:218]: same

File: lib/main.js [hunk 4]
```diff
@@ -1,87 +1,73 @@
...
-      type = TYPES[i];
-      if (typeof type.detect === 'function')
-        matched = type.detect(parsed);
-      else
-        matched = type.detect.test(parsed[0]);
-      if (matched)
-        break;
+    const parsed = parseParams(headers['content-type'])
+    let matched; let type
+    for (let i = 0; i < TYPES.length; ++i) {
```
Comment (kibertoad) [inline lib/main.js:37]: same

File: lib/types/urlencoded.js [hunk 5]
```diff
@@ -1,208 +1,189 @@
...
+  const limits = cfg.limits
+  const parsedConType = cfg.parsedConType
+  this.boy = boy
+
+  this.fieldSizeLimit = getLimit(limits, 'fieldSize', 1 * 1024 * 1024)
+  this.fieldNameSizeLimit = getLimit(limits, 'fieldNameSize', 100)
+  this.fieldsLimit = getLimit(limits, 'fields', Infinity)
+
+  let charset
+  for (let i = 0, len = parsedConType.length; i < len; ++i) {
```
Comment (kibertoad) [inline lib/types/urlencoded.js:19]: same

File: deps/dicer/lib/HeaderParser.js (hunk 2 above)
Comment (Uzlopak) [reply deps/dicer/lib/HeaderParser.js:73]: standard, what are you doing...

File: deps/streamsearch/sbmh.js (hunk 3 above)
Comment (Uzlopak) [reply deps/streamsearch/sbmh.js]: done

**Comment by Uzlopak** [comment]
I only kept the let variables in the tests, as they are not performance critical.

File: deps/dicer/lib/Dicer.js (hunk 1 above)
Comment (Uzlopak) [reply deps/dicer/lib/Dicer.js:48]: yeah, maybe for the next release :)

**Pull request merged**